packages/blade/src/utils/metaAttribute/metaAttribute.native.ts

```diff
@@ -1,5 +1,7 @@
-const metaAttribute = (_name: string, _value: string): Record<string, string> => {
-  return {};
+const metaAttribute = ({ testID }: { name?: string; testID?: string }): Record<string, string> => {
```

Comment:
Could you also add a test for this please?

Reply:
Yup, adding tests. While adding tests to some components I realised that just spreading it as a meta attribute might not be enough since some of these components internally have their own .web and .native components and I would need to pass it further down. For example on Button we have StyledButton.native where I need to pass it down. Adding tests to all components for this

Reply:
Nice find